docs/language_concepts/00_data_types.md

```diff
@@ -202,6 +202,23 @@ fn main() {
 You can create arrays of primitive types or structs. There is not yet support for nested arrays
 (arrays of arrays) or arrays of structs that contain arrays.
 
+### Slices
+
+A slice is a dynamically-sized view into a sequence of elements, usually an array. They can be resized at runtime, but because they don't own the data, they cannot be returned from a circuit. You can treat slices as arrays without a constrained size.
```

Comment:
```suggestion
A slice is a dynamically-sized view into a sequence of elements. They can be resized at runtime, but because they don't own the data, they cannot be returned from a circuit. You can treat slices as arrays without a constrained size.
```

Don't need the "usually an array" here

Reply:
Hmm actually I'm not sure now about this suggestion. I realized you probably did this because the array section comes before and you want to express how arrays are a subtype of slices?

Reply:
Yeah, more like "how vectors come into being". But I'll accept your suggestion, looks great 👍